<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>minimum rating (1-5)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>change language</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>filter</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>maximum rating (1-5)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>558</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuhelp">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actOpen"/>
    <addaction name="actSave"/>
    <addaction name="actSaveAs"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menutable">
    <property name="title">
     <string>table</string>
    </property>
    <addaction name="add_column"/>
    <addaction name="add_row"/>
   </widget>
   <widget class="QMenu" name="menuhelp_2">
    <property name="title">
     <string>help</string>
    </property>
    <addaction name="actionabout"/>
   </widget>
   <addaction name="menuhelp"/>
   <addaction name="menutable"/>
   <addaction name="menuhelp_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actOpen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="add_column">
   <property name="text">
    <string>add column</string>
   </property>
  </action>
  <action name="add_row">
   <property name="text">
    <string>add row</string>
   </property>
  </action>
  <action name="actSave">
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="actSaveAs">
   <property name="text">
    <string>save as</string>
   </property>
  </action>
  <action name="actOpen_2">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>about</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
